{"name": "test_invalid_login_shows_error", "status": "failed", "statusDetails": {"message": "AssertionError: assert None is not None", "trace": "self = <tests.test_login.TestLogin object at 0x0000027AE8FE74D0>\ndriver = <appium.webdriver.webdriver.WebDriver (session=\"b3708815025bdb47fea4c2f557434737542a21af\")>\nwait = <selenium.webdriver.support.wait.WebDriverWait (session=\"b3708815025bdb47fea4c2f557434737542a21af\")>\n\n    def test_invalid_login_shows_error(self, driver, wait):\n        lp = LoginPage(driver, wait)\n        lp.login(\"wrong_user\", \"wrong_pass\")\n        error = lp.get_error_message()\n>       assert error is not None\nE       assert None is not None\n\ntests\\test_login.py:15: AssertionError"}, "start": 1758106128173, "stop": 1758106132661, "uuid": "bd858ef9-20f3-4752-9428-04d209750a90", "historyId": "e09b804d5fa8ae0d9600e7c83287794d", "testCaseId": "f37fcd220b67e01a4c24c167768af050", "fullName": "tests.test_login.TestLogin#test_invalid_login_shows_error", "labels": [{"name": "tag", "value": "smoke"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DB-L-174"}, {"name": "thread", "value": "11924-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}